src/input/mpegts/dvb_psi_pmt.c

```diff
@@ -359,6 +359,10 @@ dvb_psi_parse_pmt
     case 0x24:
       hts_stream_type = SCT_HEVC;
       break;
+          
+    case 0x87:
```

Comment:
What about putting it after 0x81 to follow order?

Reply:
You're right.
It can be after 0x81

Reply:
I updated this file.
Now it's OK.